packages/base/src/UI5Element.js

```diff
@@ -22,6 +22,11 @@ const metadata = {
 const DefinitionsSet = new Set();
 const IDMap = new Map();
 
+/**
+ * @class
+ * @public
+ * Base class for all UI5 Web Components
```

Comment:
Perhaps these are also good to add:

 * @constructor
 * @author SAP SE
 * @alias sap.ui.webcomponents.base.UI5Element
 * @extends HTMLElement